docs/getting-started.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started
+
+Kubeapps is a set of tools written by [Bitnami](https://bitnami.com) to super-charge your Kubernetes cluster with:
+ * Your own applications [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
+ * [Kubeless](http://kubeless.io/) - a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
+ * [SealedSecrets](https://github.com/bitnami/sealed-secrets) - a SealedSecret can be decrypted only by the controller running in the cluster and nobody else (not even the original author).
```

Comment:
We should improve this description, e.g.
"a Kubernetes controller for managing encrypted secrets that even allows you to check them into git"

Reply:
I have leave this as it was, as we have a page called components.md that has a bit more of explanation.

Reply:
Okay, I just think that this description doesn't really tell me what this is, when I should use it...
Looking at the repo README, we could change this to "Encrypt your Secret into a SealedSecret, which is safe to store - even to a public repository."

Reply:
Done!